cli/tests/pcluster/config/test_section_cluster.py

```diff
@@ -378,19 +378,13 @@ def test_cluster_section_from_file(mocker, config_parser_dict, expected_dict_par
         ("custom_awsbatch_template_url", "NONE", "NONE", None),
         # Settings
         ("scaling_settings", "test1", None, "Section .* not found in the config file"),
-        ("scaling_settings", "test1,test2", None, "is invalid. It can only contains a single .* section label"),
         ("vpc_settings", "test1", None, "Section .* not found in the config file"),
-        ("vpc_settings", "test1,test2", None, "is invalid. It can only contains a single .* section label"),
```

Comment:
Why it has been removed? (some for the other settings parameters accepting a single section label.

Reply:
The logic behind this part has been slightly changed and now the condition for error "Section .* not found" is checked before the section multiplicity. 
I added a new test for single settings but I think this part would need a better refactoring in general. There are many places that check for sections multiplicity (when adding a section, in from_definition etc.) despite I think this should be an exclusive role of validation. I'd like if we could address this as a separate patch.